dockerfiles/toolbox/1-alpine-3.15/Dockerfile

```diff
@@ -13,8 +14,7 @@ RUN `#-- well known tools` \
         tar=1.34-r0 \
         wget=1.21.2-r2 \
         yq=4.14.1-r0 \
-    `#-- kubectl` \
     && wget --progress=dot:giga https://storage.googleapis.com/kubernetes-release/release/v1.23.0/bin/linux/amd64/kubectl -O /usr/bin/kubectl \
     && chmod +x /usr/bin/kubectl
 
-COPY --from=fullstorydev/grpcurl:v1.8.6 /bin/grpcurl /bin/grpcurl
+COPY --from=grpcurl /bin/grpcurl /bin/grpcurl
```

Comment:
Are you sure this is desired? By doing this you'll have no more control on the version included in the image.

Reply:
oh yes, that's right. 👍